test/browser/spec/ol/layer/Tile.test.js

```diff
@@ -40,6 +40,8 @@ describe('ol/layer/Tile', function () {
       layer = new TileLayer({
         source: new XYZ({
           url: 'spec/ol/data/osm-0-0-0.png',
+          maxZoom: 0,
+          interpolate: false,
```

Comment:
This affects the other tests within this `describe()` function. Can you please change the new test to create a TileLayer with these settings, instead of changing it for all tests here?